mdpow/tests/test_analysis.py

```diff
@@ -62,7 +65,11 @@ def fep_benzene_directory(tmpdir_factory):
 class TestAnalyze(object):
     def get_Gsolv(self, pth):
         gsolv = pth.join("FEP", "water", "Gsolv.fep")
-        G = pickle.load(gsolv.open())
+        if sys.version_info.major == 2:
+            G = pickle.load(gsolv.open())
+        elif sys.version_info.major == 3:
+            with open(gsolv, 'rb') as f:
+                G = pickle.load(f, encoding='latin1')
```

Comment:
That's freaky... is this because we are using very old pickle files in our test suite or is this a common workaround? Add a comment here for future coders.